src/Integration.UnitTests/Suppression/SuppressedIssuesProviderTests.cs

```diff
@@ -118,26 +140,37 @@ public void GetSuppressedIssues_HasSolutionBinding_ConnectedMode_ListFromBinding
         [DataTestMethod]
         [DataRow(SolutionBindingEventType.SolutionBindingUpdated)]
         [DataRow(SolutionBindingEventType.SolutionBindingChanged)]
-        public void SolutionBindingEvent_Standalone_DoesNotRaiseSuppressionsUpdatedEvent(SolutionBindingEventType eventType)
+        public void SolutionBindingEvent_Standalone_SuppressionsEventNotRaised(SolutionBindingEventType eventType)
         {
             var eventMock = new Mock<EventHandler>();
-            testSubject.SuppressionsUpdated += eventMock.Object;
+            testSubject.SuppressionsUpdateRequested += eventMock.Object;
 
             var configuration = new BindingConfiguration(new BoundSonarQubeProject(), SonarLintMode.Standalone);
             SimulateBindingEvent(eventType, configuration);
 
             eventMock.VerifyNoOtherCalls();
         }
 
+        [DataTestMethod]
+        [DataRow(SolutionBindingEventType.SolutionBindingUpdated)]
+        [DataRow(SolutionBindingEventType.SolutionBindingChanged)]
+        public void SolutionBindingEvent_Standalone_SonarQubeIssuesProviderNotCreated(SolutionBindingEventType eventType)
+        {
+            var configuration = new BindingConfiguration(new BoundSonarQubeProject(), SonarLintMode.Standalone);
+            SimulateBindingEvent(eventType, configuration);
+
+            createProviderFunc.VerifyNoOtherCalls();
```

Comment:
Presumably we're not expecting the _SuppressionsEventRaised_ event to be raised in this case?